<ui version="4.0" >
 <class>ExecuteQuery</class>
 <widget class="QDialog" name="ExecuteQuery" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>403</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>280</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>167772</width>
    <height>167772</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Execute Query</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>7</number>
   </property>
   <property name="topMargin" >
    <number>7</number>
   </property>
   <property name="rightMargin" >
    <number>7</number>
   </property>
   <property name="bottomMargin" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>3</number>
   </property>
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Edit Query:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>508</width>
         <height>13</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="m_DBlabel" >
       <property name="font" >
        <font>
         <family>Times New Roman</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Database</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="font" >
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text" >
        <string>Results:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>408</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushBtnExecute" >
       <property name="text" >
        <string>Execute</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" >
    <widget class="QRSDBTreeWidget" name="m_pResultsTree" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="MinimumExpanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="indentation" >
      <number>15</number>
     </property>
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="5" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="m_labelStatus" >
       <property name="text" >
        <string>Status:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="spacing" >
        <number>6</number>
       </property>
       <property name="leftMargin" >
        <number>0</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>0</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="m_btnOk" >
         <property name="text" >
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item rowspan="2" row="1" column="0" >
    <widget class="QTextEdit" name="plainTextEdit" />
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRSDBTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>rsdbtreewidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_btnOk</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
